examples/mnist_denoising_autoencoder.py

```diff
@@ -0,0 +1,120 @@
+'''Trains a denoising autoenconder on MNIST dataset.
+
+Gets val_loss=0.02 after 1 epoch.
```

Comment:
This is not useful information. Please replace with a few sentences about what is a denoising autoencoder and how it's setup.

Reply:
Fixed. More informative documentation added.